azure-pipelines/build/linux/du/templates/doclient-lite-docker-steps.yml

```diff
@@ -14,14 +14,14 @@ steps:
   displayName: Login to ACR
   inputs:
     command: login
-    containerRegistry: doclientcontainerregistry
+    containerRegistry: doclient-dockercontainerregistry-buildpipeline
```

Comment:
worth adding a comment that this is the name of the service connection within Azure which grants the pipelines access to the container registry?